curr_scraper/apps/api_caller/models.py

```diff
@@ -1,3 +1,19 @@
 from django.db import models
 
-# Create your models here.
+
+class Currency(models.Model):
+    code = models.TextField()
+    description = models.TextField()
+
+    class Meta:
+        app_label = "api_caller"
```

Comment:
Please add following `class Meta` attributes:
**verbose_name** - string representation of the class
**verbose_name_plural** - string representation of the class in plural form